extensions/cornerstone/src/commandsModule.ts

```diff
@@ -127,6 +127,21 @@ function commandsModule({ servicesManager, commandsManager }) {
         cstUtils.getAnnotationNearPoint(element, canvasCoordinates)
       );
     },
+    getNearbyAnnotation({ element, canvasCoordinates }) {
+      const nearbyToolData = actions.getNearbyToolData({
+        nearbyToolData: null,
+        element,
+        canvasCoordinates,
+      });
+
+      const isAnnotation = toolName =>
+        toolName !== 'Crosshairs' && toolName !== 'ReferenceLines';
```

Comment:
This should be a static set of tools which aren't annotatations, and exported by the module so it can be updated with other non-annotation tools.  Longer term, having the tool class itself export a isAnnotation method would be much better.  Having it hardcoded is really ugly.